--- linux_net.py	2015-06-11 23:23:44.407012782 +0800
+++ linux_net.py.new	2015-06-11 23:24:05.175013034 +0800
@@ -1608,6 +1608,7 @@
             # bridge, which will either be the vlan interface, or a
             # physical NIC.
             _execute('ip', 'link', 'set', bridge, 'up', run_as_root=True)
+            _execute('ip', 'link', 'set', bridge, 'promisc', 'on', run_as_root=True)
 
         if interface:
             LOG.debug('Adding interface %(interface)s to bridge %(bridge)s',
